Makefile

```diff
@@ -36,9 +30,9 @@ lint-flake8:
 
 lint: lint-black lint-pyright lint-flake8
 
-run-tests: guard-BASE_URL
+run-tests:
 	echo "Running Regression Tests"
-	poetry run python ./runner.py
+	poetry run python ./runner.py --product=EPS-FHIR --env=INT
```

Comment:
I'm not too sure about hard-coding the product and env here.

How about leaving the guard in, but using it for $PRODUCT and $ENV?